### PR TITLE
[Auth] use new method to display errors, add spinner for last onsubmit

### PR DESCRIPTION
**Pull request by ansor4**
fixes GROW-855

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 4be43cc66482212369a30a74ab930554bd20a185

https://deploy-preview-1142--artsy-reaction.netlify.com

**Comment by eessex** [comment]
👏 Exciting!  Minor comment -- the commit message must be matched to our `sematic-release` format to trigger a new release -- https://github.com/artsy/reaction/#commits-and-deployments

File: src/Components/Authentication/Mobile/LoginForm.tsx [hunk 1]
```diff
@@ -20,9 +20,19 @@ import { FormProps } from "../Types"
...
-  render() {
+  showError = status => {
     const { error } = this.props
+    if (error) {
+      return <Error show>{error}</Error>
+    }
+
+    if (status && !status.success) {
+      return <Error show>{status.error}</Error>
+    }
```
Comment (eessex) [inline src/Components/Authentication/Mobile/LoginForm.tsx]: This could use a test

**Comment by l2succes** [comment]
Congrats on your first reaction PR! 🎉Will merge in once the tests are in. You also need to prefix one of your commit messages with `[PATCH auth]` to trigger a new build (for now)

**Pull request merged**